docs/gameplay-sessions/gameplay-sessions-reference.mdx

```diff
@@ -12,19 +12,31 @@ technical overview of the Gameplay Sessions system, as well as more technical de
 
 ### Folder and File Structure
 
-When a Gameplay Session is recorded, all data is stored on disk within two folders:
+When a Gameplay Session is recorded, all data is stored on disk within two folders inside the following parent
+directory (which we will reference as the `sessionDirectory`):
 
-* `~/unity_videos/{Application.productName}/run_{#}/screenshots` - A folder with screenshots for each recorded tick of
+```
+~/unity_videos/{Application.productName}/{replay|recording}_{date}_{time}_{currentSessionId}_{referenceSessionId}
+```
+
+`currentSessionId` is a unique ID for this particular recording, and `referenceSessionId` is the ID of the Gameplay Session
+that this recording was simulated from (or equal to `currentSessionId` if this is a completely new recording not played from an 
```

Comment:
this line is misleading... `referenceSessionId` simply doesn't exist for 'recording'

Reply:
@RG-nAmKcAz I've went and updated this to hopefully explain in detail what this look like.